test/unit/test-runtime.js

```diff
@@ -1021,28 +1021,35 @@ describes.realWin(
       let importDoc;
       let hostElement;
       let ampdoc;
+      let shadowDocOptions;
 
       beforeEach(() => {
         deactivateChunking();
         clock = sandbox.useFakeTimers();
         hostElement = win.document.createElement('div');
         importDoc = win.document.implementation.createHTMLDocument('');
         importDoc.body.appendChild(win.document.createElement('child'));
-        const shadowRoot = createShadowRoot(hostElement);
-        ampdoc = new AmpDocShadow(win, docUrl, shadowRoot);
+        createShadowRoot(hostElement);
+        ampdoc = null;
 
         ampdocServiceMock
           .expects('installShadowDoc')
           .withExactArgs(
             docUrl,
-            sinon.match(arg => arg == getShadowRoot(hostElement))
+            sinon.match(arg => arg == getShadowRoot(hostElement)),
```

Comment:
change from `mock` to `spy` or `stub` might make the tests easier.

Reply:
done